packages/app-store/sendgrid/api/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as add } from "./add";
+export { default as check } from "./check";
```

Comment:
Couldn't use the defaultResponder and defaultHandler as that was what was generating the `Cannot access '__WEBPACK_DEFAULT_EXPORT__' before initialization` error.

Reply:
How was `defaultHandler` being used before?

Reply:
It’s here: https://github.com/calcom/cal.com/pull/5501/commits/385208a12e2c5e8b4cfae5e8eeaa639cc5d306a4